### PR TITLE
Feature/cycling alternatives12094

### PR DESCRIPTION
**Pull request by JulietaMarcos**
Updated routing API docs for: https://github.com/skedgo/skedgo-java/pull/1895

### CODE REVIEW

File: docs/specs/tripgo.swagger.yaml [hunk 1]
```diff
@@ -540,6 +540,23 @@ paths:
           in: query
           description: Boolean if routing result should include only the best trip found for query.
           default: false
+        - name: cyclingAlt
+          type: string
```
Comment (nighthawk) [inline docs/specs/tripgo.swagger.yaml]: Would it make sense to turn this into an `array` instead, e.g., to provide `cyclingAlt=best&cyclingAlt=friendliest&shortest`. Would make it easier to add other alternatives down the track, and allow asking for any combinations of those.

Comment (JulietaMarcos) [reply docs/specs/tripgo.swagger.yaml]: Yes... that crossed my mind too, I was undecided, I just thought the most common use would be `all`, or `best`, so I leaned towards the other option. But I agree that an `array` is more flexible and easier to add other alernatives.  I need to change the implementation a bit to support that, but it's not a big change. Should I go ahead, @nighthawk ?

Comment (JulietaMarcos) [reply docs/specs/tripgo.swagger.yaml]: Also, could we still keep `all` as a shortcut?

Comment (nighthawk) [reply docs/specs/tripgo.swagger.yaml]: Sounds good. Don’t worry changing it to an array yet.

File: docs/specs/tripgo.swagger.yaml [hunk 2]
```diff
@@ -540,6 +540,23 @@ paths:
...
+          enum:
+            - best
+            - shortest
+            - friendliest
+            - all
+            - allButShortest
```
Comment (nighthawk) [inline docs/specs/tripgo.swagger.yaml]:
```suggestion
```

Let’s not document that “all but shortest” publicly as it’s a temporary solution until this is turned into an array.

**Pull request merged**